sonic_y_cable/y_cable_base.py

```diff
@@ -0,0 +1,589 @@
+"""
+    y_cable_base.py
+
+    Base class for implementing Y-Cable specific functionality in SONiC.
+    This is the base class of sonic_y_cable implentation. A vendor specific 
+    implementation of YCable needs to inherit from this class.
+    Classes derived from this class provide the ability to interact
+    with a vendor specific Y-Cable
+"""
+
+
+class YCableBase(object):
+
+    # definitions of targets for getting the various fields/cursor
+    # equalization parameters from the register spec
+    # the name of the target denotes which side MCU
+    # values will be retreived on the Y-Cable
+
+    TARGET_UNKNOWN = -1
+    TARGET_NIC = 0
+    TARGET_TORA = 1
+    TARGET_TORB = 2
```

Comment:
May be better for readability if these are `TARGET_TOR_A` and `TARGET_TOR_B` instead

Reply:
fixed